server/commands/battle.py

```diff
@@ -0,0 +1,597 @@
+        c.battle.selected_move = -1
+        c.target = None
+    client.area.fighters = []
+    client.area.battle_started = False
+    client.send_ooc("The battle has been refreshed!")
+
+
+def ooc_cmd_atk(client, arg):
+    """
+    This command will let you use a move during a battle!
+    Heal and AttAll moves don't need a target!
+    Usage: /atk MoveName Target_ID
+    """
+    if client.battle is not None:
+        if client in client.area.fighters:
+            if client.battle.selected_move == -1:
+                args = arg.split(" ")
+                moves_list = [move.name for move in client.battle.moves]
+                if args[0].lower() in moves_list:
+                    move_id = moves_list.index(args[0].lower())
+                    if len(args) == 2:
+                        fighter_id_list = {c.id: c for c in client.area.fighters}
+                        if int(args[1]) in fighter_id_list:
+                            client.battle.target = fighter_id_list[int(args[1])]
+                            client.battle.selected_move = move_id
+                            client.area.num_selected_move += 1
+                            client.send_ooc(f"You have choosen {args[0].lower()}")
+                        else:
+                            client.send_ooc("Your target is not in the fighter list")
+                    elif "heal" in client.battle.moves[move_id]["Effects"]:
+                        client.battle.target = client
+                        client.battle.selected_move = move_id
+                        client.area.num_selected_move += 1
+                        client.send_ooc(f"You have choosen {args[0].lower()}")
+                    elif "atkall" in client.battle.moves[move_id]["Effects"]:
+                        client.battle.target = "all"
+                        client.battle.selected_move = move_id
+                        client.area.num_selected_move += 1
+                        client.send_ooc(f"You have choosen {args[0].lower()}")
+                    else:
+                        client.send_ooc("Not enough argument to attack")
+                    if client.area.num_selected_move == len(client.area.fighters):
+                        client.area.fighters = start_battle_animation(client.area)
+                        client.area.num_selected_move = 0
+                        if not client.area.battle_started:
+                            client.area.battle_started = True
+                        if len(client.area.fighters) < 2:
+                            client.area.fighters = []
+                            client.area.battle_started = False
+                else:
+                    client.send_ooc("There is no move with this name!")
+            else:
+                client.send_ooc("You already selected a move!")
+        else:
+            client.send_ooc("You are not ready to fight!")
+    else:
+        client.send_ooc("You have to choose a fighter first!")
```

Comment:
if if if if else else else else staircase is very hard to read, please revise

Reply:
Done!